src/backend/distributed/master/master_truncate.c

```diff
@@ -79,7 +79,7 @@ citus_truncate_trigger(PG_FUNCTION_ARGS)
 
 		if (MultiShardConnectionType == SEQUENTIAL_CONNECTION)
 		{
-			ExecuteModifyTasksSequentiallyWithoutResults(taskList, CMD_UTILITY);
+			ExecuteModifyTasksSequentiallyWithoutResults(taskList, MODLEVEL_NONE);
```

Comment:
This reads a little strangely, since we're clearly doing a modification here. I wonder whether we should have a MODIFY_LEVEL_DDL to indicate that this is a non-commutative modification, but we don't need special locks because it takes an AccessExclusiveLock on the table.